From 82221c364df13c8aed6b0e917575f46f04390f29 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Sat, 13 Oct 2018 12:02:40 +0530
Subject: [PATCH 0176/1851] dmaengine: xilinx_frmbuf: Fix compilation errors
 when kernel config doesn't have Framebuffer

commit 4d08696716fb0c8d36711064130442bd26ddaf2b from
https://github.com/Xilinx/linux-xlnx.git

Framebuffer is enabled in the default xilinx_zynqmp_defconfig.
This patch fixes compilation errors when Framebuffer is not configured
in the kernel.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/linux/dma/xilinx_frmbuf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/dma/xilinx_frmbuf.h b/include/linux/dma/xilinx_frmbuf.h
index 7aca4b86d554..3587998da102 100644
--- a/include/linux/dma/xilinx_frmbuf.h
+++ b/include/linux/dma/xilinx_frmbuf.h
@@ -174,7 +174,7 @@ static inline int xilinx_xdma_get_v4l2_vid_fmts(struct dma_chan *chan,
 
 static inline int xilinx_xdma_get_fid(struct dma_chan *chan,
 				      struct dma_async_tx_descriptor *async_tx,
-				      u32 *fid);
+				      u32 *fid)
 {
 	return -ENODEV;
 }
-- 
2.31.1

